static/templates/default_language_modal.hbs

```diff
@@ -6,14 +6,13 @@
     </div>
     <div class="modal-body">
         <p>
+            {{#tr this}}
             The following is a few of the languages we have started to
             support or are hoping to support in the near future. We would
-            love help translating; if you're interested in helping, make an
-            account at <a href='https://www.transifex.com/zulip/zulip/'>
-            https://www.transifex.com/zulip/zulip/</a>, and sign up for a
-            language! You can also request any language we don't currently
-            have. It only takes a few hours to translate the most important
-            parts of the app.
+            love help translating; if you're interested in helping, follow
+            <a target="_blank" href="__guidelines_link__">the guidelines</a>.
```

Comment:
@andersk suggestion on fixing this? I tried converting it to `{guidelies_link}` as per your recent changes but I get an error message I don't understand, probably something very easy for you to fix.

Reply:
<img width="1181" alt="Screenshot 2021-04-14 at 3 28 57 PM" src="https://user-images.githubusercontent.com/25124304/114692304-233e1c80-9d36-11eb-97f3-b562047a9f51.png">

Reply:
You can no longer include HTML directly in a translated string, except the very simple tags enumerated in `default_html_elements` (`<b>`, `<code>`, `<em>`, `<i>`, `<kbd>`, `<p>`, `<strong>`, all with no attributes). This is a feature, because we don’t want to expose HTML details to translators.

To insert more complicated markup, what we do now is define a custom HTML element locally to the translation. I’ve named most of them `z-link`, which you can grep for to find examples in both JavaScript:

https://github.com/zulip/zulip/blob/06af1715cbc337f4e252fe83379b1835d2520d04/static/js/compose.js#L655-L664

and Handlebars:

https://github.com/zulip/zulip/blob/06af1715cbc337f4e252fe83379b1835d2520d04/static/templates/invitation_failed_error.hbs#L10-L13

Reply:
Thanks for the detailed reply. Fixed!